docs/pages/components/tables.vue

```diff
@@ -77,6 +77,27 @@ In order to use the table, you need give `rows` and `columns` data to `KtTable`
 
 </ShowCase>
 
+## Lot of columns 
+
+Some table with a lot of columns will need to handle that overflow.
+We have two way of making the columns fits: `isScrollable` props that will make the whole table 
+horizontally scrollable.
+
+The other method is using the `kt-table--auto-layout` class that will use auto layout and will
```

Comment:
Alternatively, you can use the `kt-table--auto-layout` class to automatically break words to make the content fit into the table.